src/components/note/__internal__/note.component.js

```diff
@@ -49,7 +57,7 @@ const Note = ({
   };
 
   return (
-    <StyledNote width={width} {...rest} data-component="note">
+    <StyledNote width={width} {...marginProps} {...rest} data-component="note">
```

Comment:
By spreading `rest` here you are automatically spreading the `marginProps`
```suggestion
    <StyledNote width={width} {...rest} data-component="note">
```

Reply:
Changed as requested. That should be sorted now 👍🏻